app/javascript/app/components/header/header.js

```diff
@@ -1,3 +1,26 @@
-import Component from './header-component';
+import { withProps } from 'recompose';
 
-export default Component;
+import HeaderComponent from './header-component';
+
+function importAllImagesFromFolder(r) {
+  const images = {};
+  r.keys().map(item => {
+    images[item.replace('./', '').replace('.jpg', '')] = r(item);
+    return null;
```

Comment:
Do you need this return. Can you not use forEach instead of map to remove need for return null